docs/docs/caching.md

```diff
@@ -49,14 +55,5 @@ The following plugins have been created:
 
 When deploying with Now, follow the instructions in the [Now documentation](https://zeit.co/guides/deploying-gatsby-with-now#bonus:-cache-your-gatsby-assets).
 
----
-
-<sup>
-  1
-</sup> You can use 'no-cache' instead of 'max-age=0, must-revalidate'. Despite what the name might imply, 'no-cache' permits a cache to serve cached content as long as it validates the cache freshness first.
-<sup>
-  [2][3]{" "}
-</sup> In either case, clients have to make a round trip to the origin server on each request. However, if you are correctly utilizing ETags or Last-Modified validation you will avoid downloading assets when the cached copy is still valid (e.g. the file hasn't changed on the origin server since it was cached).
-
-[2]: https://tools.ietf.org/html/rfc7234#section-5.2.2.1
-[3]: https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching#no-cache_and_no-store
+[1]: https://tools.ietf.org/html/rfc7234#section-5.2.2.1
```

Comment:
I ended up fixing #21819 because it `<!-- lint ignore` and `gatsby-plugin-mdx` don't play nice with each other.